src/NodaTime/Calendars/Wondrous/WondrousCalendarHelper.cs

```diff
@@ -0,0 +1,2204 @@
+using JetBrains.Annotations;
+using NodaTime.Utility;
+using System;
+using System.Collections.Generic;
+
+namespace NodaTime.Calendars.Wondrous
+{
+    /// <summary>
+    /// Helper and extension methods for the Wondrous Calendar.
+    /// </summary>
+    /// <remarks>
+    /// The Wondrous Calendar Helper is in a distinct namespace so that the Extensions are only available when this
+    /// namespace is included.
```

Comment:
included => imported (that's how using directives are typically described)

Reply:
Sure